### PR TITLE
MGMT-18955: Add 4.18 RHCOS / OCP images

### PR DESCRIPTION
**Pull request by danmanor**
## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [x] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [ ] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: data/default_os_images.json [hunk 1]
```diff
@@ -184,5 +184,29 @@
...
+    {
+        "openshift_version": "4.18",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/4.18.0-ec.1/rhcos-4.18.0-ec.1-x86_64-live.x86_64.iso",
+        "version": "418.94.202409050217-0"
+      },
+      {
+        "openshift_version": "4.18",
+        "cpu_architecture": "arm64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/pre-release/4.17.0-ec.3/rhcos-4.17.0-ec.3-aarch64-live.aarch64.iso",
```
Comment (gamli75) [inline data/default_os_images.json:197]: there are no available RHCOS versions for ARM and other architectures?

Comment (danmanor) [reply data/default_os_images.json:197]: Yes I had a mixup, nice catch thank you

Comment (danmanor) [reply data/default_os_images.json]: Fixed

**Review by gamli75 — APPROVED** [review]

**Review by gamli75 — APPROVED** [review]

**Pull request merged**